client-templates/gitlab/accept.json.sample

```diff
@@ -0,0 +1,35 @@
+{
+  "private":
+  [
+    {
+      "//": "list the user's projects",
+      "method": "GET",
+      "path": "/api/v3/projects",
+      "origin": "https://${GITLAB_API}"
+    },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/api/v3/projects/:project/repository/files",
+      "origin": "https://${GITLAB_API}",
+      "validParams": [
+        {
+          "param": "file_path",
+          "value": "package.json"
+        },
+        {
+          "param": "file_path",
+          "value": "Gemfile.lock"
+        },
+        {
+          "param": "file_path",
+          "value": ".snyk"
+        },
+        {
+          "param": "ref",
+          "value": "master"
+        }
+      ]
```

Comment:
I suggest a different approach, where each param has a list of possible values for filtering, like so:
```
"validParams": {
  "file_path": ["package.json", "Gemfile.lock", ".snyk"]
}
```

This way you can later filter by possible values in code, see next comment.